src/main/webapp/dashboard-manager.js

```diff
@@ -101,14 +100,18 @@ class DashboardManager {
       this.data.addColumn('string', value);
     }
     this.data.addColumn('number', 'Devices');
-    this.data.addColumn('string', 'deviceIds');
     this.data.addColumn('string', '');
+    this.data.addColumn('string', 'deviceIds');
+    this.data.addColumn('string', 'serialNumbers');
+    this.data.addColumn('number', 'row');
 
     let aggregationLoader = new Loading(this.fetchAndPopulateAggregation.bind(this), false);
     await aggregationLoader.load();
 
-    this.COLS.DEVICE_ID = this.data.getNumberOfColumns() - 2;
-    this.COLS.DEVICE_COUNT = this.data.getNumberOfColumns() - 3;
+    this.COLS.ROW_NUMBER = this.data.getNumberOfColumns() - 1;
+    this.COLS.SERIAL_NUMBERS = this.data.getNumberOfColumns() - 2;
+    this.COLS.DEVICE_IDS = this.data.getNumberOfColumns() - 3;
+    this.COLS.DEVICE_COUNT = this.data.getNumberOfColumns() - 5;
```

Comment:
maybe add a comment or make these named constant variables to explain where they're coming from

Reply:
The column numbering is definitely confusing and a weak point in the front end's design.  I've added a comment directing the reader to an explanation of where the numbers come from. (I'm opting not to make them constant variables because 1. it would not help understand how the numbers were chosen and 2. the numbers are subject to change if our table layout changes in the future)